_security-plugin/configuration/multi-auth.md

```diff
@@ -0,0 +1,132 @@
+
+To specify the multiple authentication types as options during sign-in, add the `opensearch_security.auth.type` setting to the `opensearch_dashboards.yml` file and enter multiple types as values. When more than one authentication type is added to the setting, the Dashboards sign-in window recognizes multiple types and adjusts to accommodate the sign-in options.
+
+When setting up Dashboards to provide multiple authentication options, basic authentication is always required as one of the values for the setting.
+{: .note }
+
+For single option sign-in, the authentication type is specified by adding a single type to the setting.
+
+```yml
+opensearch_security.auth.type: "openid"
+```
+
+For multiple option authentication, add values to the setting as an array separated by commas. As a reminder, Dashboards currently supports a combination of basic authentication, OpenID Connect, and SAML as a valid set of values. In the setting, these values are expressed as `"basicauth"`, `"openid"`, and `"saml"`.
+
+```yml
+opensearch_security.auth.type: ["basicauth","openid"]
+opensearch_security.auth.multiple_auth_enabled: true
+```
+
+```yml
+opensearch_security.auth.type: ["basicauth","saml"]
+opensearch_security.auth.multiple_auth_enabled: true
+```
+
+```yml
+opensearch_security.auth.type: ["basicauth","saml","openid"]
+opensearch_security.auth.multiple_auth_enabled: true
+```
+
+When the `opensearch_security.auth.type` setting contains `basicauth` and one other authentication type, the sign-in window appears as in the example below.
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/OneOptionWithoutLogo.png" alt="Basic authentication and one other type in the sign-in window" width="350">
+
+With all three valid authentication types specified, the sign-in window appears as in the following example:
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/TwoOptionWithoutLogo.png" alt="All three authentication types specified in the sign-in window" width="350">
+
+## Customizing the sign-in environment
+
+In addition to the essential sign-in settings for each authentication type, you can configure additional settings in the `opensearch_dashboards.yml` file to customize the sign-in window so that it clearly represents the options that are available. For example, you can replace the label on the sign-in button with the name and icon of the IdP. Use the settings below to change the look and feel of the different options.
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/TwoOptionWithLogo.png" alt="Multi-option sign-in window with with some customization" width="350">
+
+### Basic authentication settings
+
+The settings below are used to customize the basic username and password sign-in button.
+
+Setting | Description
+:--- | :--- |:--- |:--- |
+`opensearch_security.ui.basicauth.login.buttonname` |  Display name for the login button. "Log in" by default.
```

Comment:
Sorry for the confusion, basicauth authentication does not have the setting to customize the login buttonname - `opensearch_security.ui.basicauth.login.buttonname`. Please help remove it from the table.

Reply:
Removed this setting.